### ISSUE TITLE
运行horvord与dpgen的时候，出现单机多卡负载不均衡情况

### ISSUE THREAD

**Issue by zhoujingyu13687306871** [deepmodeling/deepmd-kit#1452]
我在运行脚本时候，如下：

#!/bin/bash
module load anaconda/2020.11
source activate deepmd
module load cudnn/8.2.1_cuda11.x cuda/11.2 nccl/2.11.4-1_cuda11.2 openmpi/4.1.1
export CUDA_VISIBLE_DEVICES=0,1

horovodrun -np 2 dp train --mpi-log=workers /HOME/scz1056/deep/workspace/deepmd-kit/examples/water/se_e2_a/input.json 1>erro 2>out

提交后，GPU卡运行情况如下：
<img width="558" alt="企业微信截图_16431152091705" src="https://user-images.githubusercontent.com/47057417/150981187-e25e767d-ef3c-4331-b414-6e895ab386ed.png">

input.json内容如下：
  1 {
  2         "_comment": " model parameters",
  3         "model": {
  4                 "type_map": [
  5                         "O",
  6                         "H"
  7                 ],
  8                 "descriptor": {
  9                         "type": "se_e2_a",
 10                         "sel": [
 11                                 46,
 12                                 92
 13                         ],
 14                         "rcut_smth": 0.50,
 15                         "rcut": 6.00,
 16                         "neuron": [
 17                                 25,
 18                                 50,
 19                                 100
 20                         ],
 21                         "resnet_dt": false,
 22                         "axis_neuron": 16,
 23                         "seed": 1,
 24                         "_comment": " that's all"
 25                 },
 26                 "fitting_net": {
 27                         "neuron": [
 28                                 240,
 29                                 240,
 30                                 240
 31                         ],
 32                         "resnet_dt": true,
 33                         "seed": 1,
 34                         "_comment": " that's all"
 35                 },
 36                 "_comment": " that's all"
 37         },
 38         "learning_rate": {
 39                 "type": "exp",
 40                 "decay_steps": 5000,
 41                 "start_lr": 0.001,
 42                 "stop_lr": 3.51e-8,
 43                 "_comment": "that's all"
 44         },
 45         "loss": {
 46                 "type": "ener",
 47                 "start_pref_e": 0.02,
 48                 "limit_pref_e": 1,
 49                 "start_pref_f": 1000,
 50                 "limit_pref_f": 1,
 51                 "start_pref_v": 0,
 52                 "limit_pref_v": 0,
 53                 "_comment": " that's all"
 54         },
 55         "training": {
 56                 "training_data": {
 57                         "systems": [
 58                                 "/HOME/scz1056/deep/workspace/deepmd-kit/examples/water/data/data_0/",
 59                                 "/HOME/scz1056/deep/workspace/deepmd-kit/examples/water/data/data_1/",
 60                                 "/HOME/scz1056/deep/workspace/deepmd-kit/examples/water/data/data_2/"
 61                         ],
 62                         "batch_size": "auto",
 63                         "_comment": "that's all"
 64                 },
 65                 "validation_data": {
 66                         "systems": [
 67                                 "/HOME/scz1056/deep/workspace/deepmd-kit/examples/water/data/data_3"
 68                         ],
 69                         "batch_size": 1,
 70                         "numb_btch": 3,
 71                         "_comment": "that's all"
 72                 },
 73                 "numb_steps": 1000000,
 74                 "seed": 10,
 75                 "disp_file": "lcurve.out",
 76                 "disp_freq": 100,
 77                 "save_freq": 1000,
 78                 "_comment": "that's all"
 79         },
 80         "_comment": "that's all"
 81 }

@njzjz

**Comment by AnguseZhang** [comment]
Please use English to recreate an issue. This issue is declined.
